encall.sh

```diff
@@ -1,29 +1,95 @@
-#!/bin/bash
+#!/bin/sh
+#cito M:755 O:0 G:0 T:/usr/local/bin/crypto
```

Comment:
Feel free to remove this. Admittedly, it's more of a plug. :P But hey, it is pretty convenient. lol

Reply:
hi @terminalforlife , you're back with new great update man, it's a completely different from the previous version , I tested it everything looks great except of that if you run the script without any argument it should show the usage (help) massage, but instead it will try to execute crypto without `-e` or `-d` and an error will appear !.
Also when i use `-d` it will try to decrypt even the files that doesn't end with `.hacklab` (not an encrypted files).

I really appreciate your time and this work , it's great chance for me to learn from you :)